### ISSUE TITLE
A1_imageRetrieval.py Found ("ERROR: image path does not exist.")

### ISSUE THREAD

**Issue by thinktwice812** [Azure/ImageSimilarityUsingCntk#2]
Hi Bros
When I run A1_imageRetrieval.py That found ("ERROR: image path does not exist.").
There is not exists 95.jpg in data\fashionTexture\striped.
So I have to create it.

**Comment by radinhamidi** [comment]
Yes!, Also you can add an input line of code so you can interact with the code. 
Like:
queryImgPath = input("Please enter target image path:")
